### PR TITLE
Update docs for variable precision timestamp support in Hive connector

### PR DESCRIPTION
**Pull request by aalbu**

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 1]
```diff
@@ -288,6 +288,11 @@ Property Name                                      Description
 ``hive.orc.time-zone``                             Sets the default time zone for legacy ORC files that did	JVM default
                                                    not declare a time zone.
 
+``hive.timestamp-precision``                       Specifies the precision to be used for columns of type 	    ``MILLISECONDS``
+                                                   ``timestamp``. Possible values are ``MILLISECONDS``,
+                                                   ``MICROSECONDS`` and ``NANOSECONDS``. Only
+                                                   ``MILLISECONDS`` is supported for write operations.
```
Comment (findepi) [inline presto-docs/src/main/sphinx/connector/hive.rst:294]: "Writes are not supported when the value is other than MILLISECONDS." ?

**Review by findepi — APPROVED** [review]

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 2]
```diff
@@ -288,6 +288,11 @@ Property Name                                      Description
 ``hive.orc.time-zone``                             Sets the default time zone for legacy ORC files that did	JVM default
                                                    not declare a time zone.
 
+``hive.timestamp-precision``                       Specifies the precision to be used for columns of type 	    ``MILLISECONDS``
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/hive.rst:291]: the precision to use for ..

File: presto-docs/src/main/sphinx/connector/hive.rst [hunk 3]
```diff
@@ -288,6 +288,11 @@ Property Name                                      Description
...
+                                                   ``timestamp``. Possible values are ``MILLISECONDS``,
+                                                   ``MICROSECONDS`` and ``NANOSECONDS``. Writes are not
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/hive.rst:293]: Write operations are only supported for ``MILLISECONDS``.

**Review by mosabua — APPROVED** [review]
Nice and concise..,

**Comment by mosabua** [comment]
Please merge before 342 is cut

**Pull request merged**